content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -2822,10 +2942,16 @@ It returns the initial state of the current Agent.
 | `application.name`                  | No       | `string`  | Application name                                                                                                                 |
 | `application.version`               | No       | `string`  | Application version                                                                                                              |
 | `away` __*__                        | No       | `bool`    | When `true`, the connection is set to the `away` state. Defaults to `false`.                                                     |
+| `customer_push_level`               | No       | `string`  | Possible values: `my`, `engaged`, `online` __**__. Defaults to `my`.                                                             |
```

Comment:
defaults to `my` if login creates the first session; if the session already exists then the current `customer_push_level` won't change